linera-core/src/client/mod.rs

```diff
@@ -1375,9 +1392,22 @@ where
                 return;
             }
         }
+        for (chain_id, height) in received_certificates.other_sender_chains {
+            if let Err(error) = self
+                .client
+                .local_node
+                .wait_for_outgoing_messages(chain_id, height)
+                .await
+            {
+                warn!(
```

Comment:
This should never happen under proper operation, right? I'm thinking whether `Err(_)` shouldn't be more sever than just a warning 🤔

Reply:
Good point, I'll make it an `error!`.